examples/miscellaneous/rascal1DemoProjects/README.md

```diff
@@ -0,0 +1,6 @@
+## Rascal 1 Demo Projects
+
+This shows examples ported from the RasCAL 1 demo projects. The following were ported
+
+- monolayerVolumeModel
+- Monolayer_8_contrasts (monolayer7Contrast.m)
```

Comment:
As discussed, some more detail here would be helpful. Explicitly pointing out that these examples show how users can port over their projects would be a good idea.